src/auth.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { getSignInUrl, getSignUpUrl, signOut } from './auth.js';
+import * as authorizationUrl from './get-authorization-url.js';
+import * as session from './session.js';
+
+const terminateSession = jest.mocked(session.terminateSession);
+
+jest.mock('./session', () => ({
+  terminateSession: jest.fn().mockResolvedValue(new Response()),
+}));
+
+describe('auth', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
Do we need to manually call `jest.clearAllMocks` if `clearMocks` is set in the `jest.config.ts`?